### DISCUSSION TITLE
DMX diagram for ESP V3.0.

### DISCUSSION THREAD

**Discussion by ganderslight** [forkineye/ESPixelStick#739] [Q&A]
I have been unable to find any diagrams for wiring a DMX output on the V3.0.  There are four pins  V+ D+ D- V- .  My thought  is that D- is pin 3 / D+ is Pin 2 and  1 is ground.  is this correct?

**Comment by onewithhammer** [comment]
Yes - correct.
See diagram here
https://docs.google.com/drawings/d/1j_HNGDBPm8GtM7F4URYRfYqPcgDAmRXGkmTZjZ-yxjA/edit?usp=sharing
Full Video here
https://www.youtube.com/watch?v=aNw_iXwn4mw
